pallets/multisig/src/lib.rs

```diff
@@ -927,16 +932,12 @@ impl<T: Config> Module<T> {
                     );
                 }
                 if proposal_details.auto_close {
-                    let approvals_needed = Self::ms_signs_required(multisig.clone());
-                    let ms_signers = Self::number_of_signers(multisig.clone());
-                    if proposal_details.rejections > ms_signers.saturating_sub(approvals_needed) {
-                        proposal_details.status = ProposalStatus::Rejected;
-                        Self::deposit_event(RawEvent::ProposalRejected(
-                            current_did,
-                            multisig.clone(),
-                            proposal_id,
-                        ));
-                    }
+                    proposal_details.status = ProposalStatus::Rejected;
+                    Self::deposit_event(RawEvent::ProposalRejected(
+                        current_did,
+                        multisig.clone(),
+                        proposal_id,
+                    ));
```

Comment:
This would make any rejection stop the proposal, even if there are more approvals.